homeassistant/components/switchbot_cloud/manifest.json

```diff
@@ -6,5 +6,5 @@
   "documentation": "https://www.home-assistant.io/integrations/switchbot_cloud",
   "iot_class": "cloud_polling",
   "loggers": ["switchbot-api"],
-  "requirements": ["switchbot-api==2.0.0"]
+  "requirements": ["switchbot-api==2.1.0"]
```

Comment:
Please split the dependency bump into a new PR as we require that the PRs are as small as possible.

Keep this PR in draft until the other one is merged. The advantage of splitting is that it allows us to ship dependencies bumps in patch releases.